types/openseadragon/index.d.ts

```diff
@@ -0,0 +1,1468 @@
+// Type definitions for OpenSeadragon 2.4
+// Project: https://openseadragon.github.io/
+// Definitions by:  Álvaro Martínez <https://github.com/alvaromartmart>
+//                  RobertYost <https://github.com/RobertYost>
+//                  Jasper Staab <https://github.com/jstaab>
+//                  Kristin Ruben <https://github.com/kristinruben>
+//                  Geoff Harper <https://github.com/geoff-harper>
+// Definitions: https://github.com/alvaromartmart/types-openseadragon
+// TypeScript Version: 3.5
+/// <reference types="node"/>
```

Comment:
I think you don't need this line at all, as there is no direct reference to Node I think

Reply:
:+1: